README.md

```diff
@@ -19,33 +19,38 @@ Questions can be asked by raising an `Issue`.
 ## How to clone repo and make changes locally
 
 ```
-  click on the clone button (green in colour).
+  click on the clone button (green in colour). This gives you a copy of the project. Its now yours to play around with
 ```
 
-- Using git on your local machine
+- Using git on your local machine. Do this to download the forked copy of this repo to your computer
 
 ```
-  git clone https://github.com/eddiejaoude/hacktoberfest-practice.git
+  git clone https://github.com/yourGithubUsername/hacktoberfest-practice.git 
 ```
 
-- switch to the cloned folder
+- switch to the cloned folder. This can be done with Gitbash or the integrated terminal in the VSCode editor
 
 ```
   cd hacktoberfest-practice
 ```
 
-- Make a new branch
+- Make a new branch. Your name would make a good branch because it's unique
 
 ```
   git checkout -b <name of new branch>
 ```
 
-- Make changes in your repo
+- Make changes in your repo by opening the README.md file and adding your full name along with a link to your github profile
 
 ```
-  touch text.txt
-  echo "Hallo world" > text.txt
-  git add text.txt
+  open ReadMe. md
+  On a new line type in ..
+  Full Name [github.com/githubUsername]
+```
```

Comment:
I think think might be better as a list, something like

- open the file `README.md`
- on a new line in alphabetical order add your name and link to you GitHub
- for example `[Name](https://github.com/your-username)`